carbonmark-api/test/routes/retirements/get.test.mocks.ts

```diff
@@ -0,0 +1,53 @@
+import { pick } from "lodash";
+import nock from "nock";
+import { GRAPH_URLS } from "../../../src/app.constants";
+import { fixtures } from "../../fixtures";
+
+const mockProfile = fixtures.firebase.profile;
+const mockRetirement = fixtures.digitalCarbon.retirement;
+
+export const expectedTransformedRetirement = {
+  ...pick(mockRetirement, [
+    "id",
+    "bridgeID",
+    "beneficiaryName",
+    "retirementMessage",
+    "retiringName",
+  ]),
+  timestamp: 1701095367,
+  retiringAddress: mockRetirement.retiringAddress.id,
+  beneficiaryAddress: mockRetirement.beneficiaryAddress.id,
+  amount: 3,
+  hash: mockRetirement.hash,
+  hasProvenanceDetails: false,
+  credit: {
+    ...pick(mockRetirement.credit, ["id", "bridgeProtocol", "vintage"]),
+    projectId: mockRetirement.credit.project.id,
+    currentSupply: 320308,
+    retired: 0,
+    crossChainSupply: 0,
+  },
+  retireeProfile: mockProfile,
+};
+
+export const mockDatabaseProfile = () =>
+  jest.mock("../../../src//utils/helpers/users.utils", () => {
+    const carbonProjectsUtils = jest.requireActual(
+      "../../../src/utils/helpers/users.utils"
+    );
+    return {
+      ...carbonProjectsUtils,
+      getProfileByAddress: jest.fn(() => {
+        return mockProfile;
+      }),
+    };
```

Comment:
Can you use nock or the mockFirebase utility instead

https://github.com/KlimaDAO/klimadao/blob/512ee69aa794e276bcd9cf0526360ff99cb11960/carbonmark-api/test/test.utils.ts#L9

Reply:
Done in 39cb34634